packages/models/src/classMap.js

```diff
@@ -89,6 +90,14 @@ export default class ClassMap {
     return this.root(CodeObjectType.DATABASE);
   }
 
+  setTemporaryRoot(id) {
+    this.roots = [this.codeObjectsById[id]];
+  }
+
+  clearTemporaryRoot() {
+    this.roots = [...this.originalRoots];
+  }
+
```

Comment:
I don't think `models` is the right place for this. This feels like render state to me, and we should limit that as best we can to the `components` package. Is there a better place for this?

Reply:
You're right, I removed that code modifications from `models` package.